helm/chaos-mesh/templates/prometheus-deployment.yaml

```diff
@@ -32,6 +32,13 @@ spec:
       {{- if .Values.prometheus.serviceAccount }}
       serviceAccount: {{ .Values.prometheus.serviceAccount }}
       {{- end }}
+      initContainers:
+        - name: data-permission-fix
+          image: busybox
+          command: ["/bin/chmod","-R","777", "/data"]
+          volumeMounts:
```

Comment:
What is this volume used for?

Reply:
The same as https://github.com/pingcap/tidb-operator/blob/62ad3e41f741b3cb0c84a05f8a47e351dd9382b4/pkg/monitor/monitor/util.go#L257:6. Fix the data permission problem